codegen/src/test/resources/software/amazon/awssdk/codegen/poet/batchmanager/test-batchmanager-async-interface.java

```diff
@@ -0,0 +1,106 @@
+package software.amazon.awssdk.services.batchmanagertest.batchmanager;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ScheduledExecutorService;
+import software.amazon.awssdk.annotations.Generated;
+import software.amazon.awssdk.annotations.SdkPublicApi;
+import software.amazon.awssdk.core.batchmanager.BatchOverrideConfiguration;
+import software.amazon.awssdk.services.batchmanagertest.BatchManagerTestAsyncClient;
+import software.amazon.awssdk.services.batchmanagertest.batchmanager.internal.DefaultBatchManagerTestAsyncBatchManager;
+import software.amazon.awssdk.services.batchmanagertest.model.DeleteRequestRequest;
+import software.amazon.awssdk.services.batchmanagertest.model.DeleteRequestResponse;
+import software.amazon.awssdk.services.batchmanagertest.model.SendRequestRequest;
+import software.amazon.awssdk.services.batchmanagertest.model.SendRequestResponse;
+import software.amazon.awssdk.utils.SdkAutoCloseable;
+
+/**
+ * Batch manager class that implements automatic batching features for a BatchManagerTest sync client. This can be
+ * created using the static {@link #builder()} method.
```

Comment:
Can we add some javadocs to explain the automatic batching features?

Reply:
Added javadocs to explain the automatic batching feature.